src/nightwatch/setup-helper.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const fs = require('fs');
+const loadJsonFile = require('load-json-file');
+const globSync = require('glob').sync;
+
+const interatePackageDirectories = (lernaJSON) => {
+  const srcFolders = [];
```

Comment:
nitpic: JavaScript is a very functional language and usually a declarative pattern is preferred over an imperative one. One sign that a function is imperative is usually the declaration of a variable at the top that is appended/modified throughout a series of operations. 

Declarative implementation:
```js
return  loadJsonFile.sync(lernaJSON).packages  // array of directories
  .map(globSync) // array of array of file paths
  .reduce((a,b) => a.concat(b), []) // Merge file paths to single array
  .map(packagePath => `${packagePath}/tests/nightwatch/`) // Get test file path
  .filter(fs.existsSync); // Filter to valid test paths
```

We shouldn't have to search sub-directories. If we do, its likely because our filter is `*-spec.js` and should actually be `**/*-spec.js`
https://github.com/nightwatchjs/nightwatch/blob/master/test/nightwatch.json#L11